test/azure/generated/AzureSpecials/autoRestAzureSpecialParametersTestClientContext.ts

```diff
@@ -26,7 +29,7 @@ export class AutoRestAzureSpecialParametersTestClientContext extends msRestAzure
    * credentials. The value is always '1234-5678-9012-3456'
    * @param [options] The parameter options
    */
-  constructor(credentials: msRest.ServiceClientCredentials, subscriptionId: string, options?: Models.AutoRestAzureSpecialParametersTestClientOptions) {
+  constructor(credentials: msRest.ServiceClientCredentials | TokenCredential, subscriptionId: string, options?: Models.AutoRestAzureSpecialParametersTestClientOptions) {
```

Comment:
Can we have the jsdocs generated above for `credentials` to add a hint on how to obtain the credentials?